### ISSUE TITLE
Member Custom Status = none

### ISSUE THREAD

**Issue by DampfWaffel** [SkyCraft78/DiSky#2]
Hello, 
me again.
- Is there a way to i.g ask discord if some user, has certain permissions? (Vixio variant https://olyno.github.io/VixioSite/#member_has_permission)
- Is there a expression for the Online Status of Bots/Users? like (Vixio variant <https://olyno.github.io/VixioSite/#online_status_of>)

and the Member Custom Status <https://skripthub.net/docs/?id=5036> always return none.<http://i.epvpimg.com/HsQ1dab.png>

Latest.log <https://pastebin.com/uhY2bTbF>
Code:
```js
discord command embed:
	prefixes: !
	trigger:
		if event-member has permission administrator:
			reply with "test %custom status of event-member%"
```
			
<http://i.epvpimg.com/483vdab.png>

**Comment by SkyCraft78** [comment]
Hi!
The `has permission` was registred as an expression and not as a condition. It will be fixed in the next update.
The `custom status` one return the **precense** and not the "real" custom status.
I'll clean up the docs and syntaxes for getting both of them with different syntaxes ^^
Thanks anyway for opening an issue, it help me make DiSky more organized <3

**Comment by SkyCraft78** [comment]
Fixed in cbdb447 & ac479f6
